.husky/organizations.ts

```diff
@@ -0,0 +1,142 @@
+import { turtle } from '@tpluscode/rdf-string'
```

Comment:
How did file get here? :) Please remove it and update `apis/core/bootstrap/organizations.ts`

Reply:
Hm could it be that husky did that somehow? Else: sorry! Should be fixed now...